dhall-yaml/src/Dhall/Yaml.hs

```diff
@@ -60,8 +61,17 @@ jsonToYaml json documents quoted =
     style (Y.SStr s)
         | "\n" `Text.isInfixOf` s =
             Right (YE.untagged, YE.Literal YE.Clip YE.IndentAuto, s)
-        | quoted =
+        | quoted || Text.all isNumberOrDateRelated s || isBoolString =
             Right (YE.untagged, YE.SingleQuoted, s)
+      where
+        isBoolString
+          | Text.length s > 5 = False
+          | otherwise =
+            case Text.toLower s of
+              "true" -> True
+              "false" -> True
+              _ -> False
```

Comment:
Could you add tests for this "bool-string" handling?

Reply:
done